contents/blog/the-posthog-array-1-42-0.md

```diff
@@ -0,0 +1,116 @@
+
+Self-hosted users must run async migrations 0005-0007 before updating to 1.42.0. 
+
+If you haven't run async migrations 0005-0007 yet, or if they aren't available on your current version,  we recommend upgrading to 1.41.4 first, _then_ running the async migrations, then upgrading to 1.42.0.
+
+## PostHog 1.42.0 release notes
+
+**Release highlights:**
+
+- [New: Area chart visualization](#new-area-chart-visualization)
+- [New: URL breakdown normalisations](#new-url-breakdown-normalisations)
+- [New: Share recordings more easily](#new-share-recordings-more-easily)
+- [New: Vote on our roadmap](#new-vote-on-our-roadmap)
+- [Boring legal bit: Policy updates](#boring-legal-bit-policy-updates)
+
+### New: Area chart visualization
+
+![area chart visualization in posthog](../images/blog/array/1-42-0-area-charts.gif)
+
+New team member Thomas Obermueller has got off to a great start by delivering the much-requested area chart visualization for PostHog insights. Now, if a normal line chart isn't  enough and a regular bar chart won't satisfy then you have an exciting third option.
+
+To [quote Thomas](https://github.com/PostHog/posthog/pull/12869) himself: "_The main use case for this is when you want to see the composition of a metric, while also having the total value as a reference e.g. breakdown of revenue by customer over time._"
+
+### New: URL breakdown normalisations
+
+![URL breakdown normalization](../images/blog/array/1-42-0-url-breakdown.gif)
+
+Are you frustrated when you breakdown trends and funnels by URLs and see four different URLs that look like: `/home/`, `/home`, `/home?`, and `/home/#`? Well, not anymore! 
+
+Now, whenever we're dealing with breakdowns by "Current URL" or "Pathname", we normalise URLs to count the same, so you'll see there's only one place like `/home`. 
+
+### New: Share recordings easily, and precisely
+
+![share recordings in posthog](../images/blog/array/1-42-0-share-recordings.gif)
+
+When you find an interesting session recording, you probably want to share it with your team. So, we've improved sharing options to make it easier. You can now share specific recordings easily, and even send teammates to an exact time in any recording!
+
+### New: Vote on our roadmap
+
+![posthog roadmap](../images/blog/array/1-42-0-roadmap.gif)
+
+Even though we're open source, we know it's not always very easy to tell what we're working on or planning for the future - there are just so many repos, issues and PRs!
+
+So, [we built a public roadmap of our major projects](/roadmap). You can use it to vote on upcoming projects, subscribe for updates and give feedback on our wildest ideas. Best of all, it's [powered by Squeak!](https://squeak.posthog.com/)
+
+### Boring legal bit: Policy updates
+
+Despite being described by our VP of Marketing and Ops as "_the least interesting thing ever_", we nevertheless need to tell you that there have been some changes to our terms and privacy policies. Don't worry, it's all good news:
```

Comment:
The photo in the mail here seems to suggest this has to do with 'disable hedgehog mode' ?

<img width="727" alt="image" src="https://user-images.githubusercontent.com/7115141/204630849-e9b5a706-dc4f-4b3a-ad7a-c3065f19de35.png">